### PR TITLE
some minor cleanups

### PR DESCRIPTION
**Pull request by giuseppe**
two tweaks I've noticed while working on a diffent issue.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: src/ctr_logging.c [hunk 1]
```diff
@@ -54,6 +54,11 @@ static char *container_name = NULL;
 static char *container_tag = NULL;
 static size_t container_tag_len;
 
+typedef struct {
+	int iovcnt;
+	struct iovec iov[WRITEV_BUFFER_N_IOV];
+} writev_buffer_t;
+
```
Comment (TomSweeneyRedHat) [inline src/ctr_logging.c]: Just curious about this move?  I know writev_buffer_t is only used currently here in this c file and probably will only ever be used here.  But did you see a compilation or other error with it in the .h file?  The move is fine, just wondering about the motivation.

**Comment by TomSweeneyRedHat** [comment]
LGTM, I do have one question to satisfy my curiosity.

**Comment by TomSweeneyRedHat** [comment]
and all happy green test buttons.

File: src/ctr_logging.c (hunk 1 above)
Comment (giuseppe) [reply src/ctr_logging.c]: no no compilation warning.  It is just to make explicit it is used only in this file and prevent in the future to have functions using write_buffer_t in other files

Comment (TomSweeneyRedHat) [reply src/ctr_logging.c]: Thanks for the 411 @giuseppe.

**Pull request merged**